Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -686,8 +687,8 @@ def top_tags_results_command():
         'ContentsFormat': formats['text'],
         'Contents': results,
         'EntryContext': {'AutoFocus.TopTagsResults(val.PublicTagName == obj.PublicTagName)': context,
-                         'AutoFocus.SessionsSearch(val.AFCookie == obj.AFCookie)': {'Status': status,
-                                                                                    'AFCookie': af_cookie}},
+                         'AutoFocus.TopTagsSearch(val.AFCookie == obj.AFCookie)': {'Status': status,
```

Comment:
why did you change the path?

Reply:
It was a mistake. The old path was only entered last week so no danger of breaking backward compatibility.